tests/Microsoft.Identity.Test.LabInfrastructure/LabServiceParameters.cs

```diff
@@ -0,0 +1,94 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Identity.Test.LabInfrastructure
```

Comment:
Could we get comments on some of these? Some of the values, like `AutoMfaOnAll`, most of the ProtectionPolicy values  are not familiar.

Reply:
also, is there a reason the names don't follow standard practice (ex. camel casing)?

---
In reply to: [328545360](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1400#discussion_r328545360) [](ancestors = 328545360)